layers/auto-completion/packages.el

```diff
@@ -269,7 +269,7 @@
         "Call `yas-expand' and switch to `insert state'"
         (interactive)
         (call-interactively 'aya-expand)
-        (evil-insert-state))
+        (unless holy-mode (evil-insert-state)))
```

Comment:
@nixmaniack I don't know if it will be needed with #4456. If I understand correctly after this hybrid mode will have an insert state (but without bindings), no?